environment_variables_guide.md

```diff
@@ -0,0 +1,89 @@
+# Environment Variables and rcfiles Guide
+
+## Environment Variables Overview
+
+Environment variables are dynamic-named values that affect the processes running on an operating system. They are used to store configuration settings, paths, and other important information. Here's a step-by-step guide on how to work with environment variables.
```

Comment:
```suggestion
Environment variables are dynamically-named values that affect the processes running on an operating system. They are used to store configuration settings, paths, and other important information. Here's a step-by-step guide on how to work with environment variables.
```

Reply:
However, in general, these docs don't need to explain this in a general way. Mommy needs an explanation for `cargo mommy` in particular!